mail_base/README.rst

```diff
@@ -0,0 +1,11 @@
+Mail Base
+=========
+
+Module contain common code for other mail modules
```

Comment:
Modules doesn't introduce new features, but make built-in mail js features extendable.